react-swipe.js

```diff
@@ -65,9 +65,16 @@
     shouldComponentUpdate: function (nextProps) {
       return (
         (this.props.slideToIndex !== nextProps.slideToIndex) ||
-        (typeof this.props.shouldUpdate !== 'undefined') && !this.props.shouldUpdate(nextProps)
+        (typeof this.props.shouldUpdate !== 'undefined') && !this.props.shouldUpdate(nextProps)||
+        (nextProps.children!==this.props.children)
```

Comment:
Because I want to reset "this.swipe = Swipe(this.getDOMNode(), this.props);".If I don't do this,the  childrens can't be display.